From 149652144606c50b703848c99e2637a4f99bdbc6 Mon Sep 17 00:00:00 2001
From: ddugger@redhat.com <ddugger@redhat.com>
Date: Thu, 22 Sep 2011 17:23:49 +0200
Subject: [PATCH 07/76] bz716261: Enable +xsave as a -cpu flag

RH-Author: ddugger@redhat.com
Message-id: <20110922172349.GB17560@sobek.n0ano.com>
Patchwork-id: 33204
O-Subject: Re: [RHEL 6.2 PATCH 7/6 V3] bz716261: Enable +xsave as a -cpu flag
Bugzilla: 716261
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Avi Kivity <avi@redhat.com>

Allows the user to specify the cpu type `-cpu +xsave' to expose just
the xsave CPUID capability to the guest.

Although not specifically required for xsave support this is an extremely
nice to have feature that requires a very simple change and would
therefore be appropriate for adding to this patch series.

Testing: Has been tested and does properly enable the XSAVE capability.

Signed-off-by: Don Dugger <donald.d.dugger@intel.com>
---
 target-i386/helper.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 target-i386/helper.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/target-i386/helper.c b/target-i386/helper.c
index b1a0ae1..2f77061 100644
--- a/target-i386/helper.c
+++ b/target-i386/helper.c
@@ -55,7 +55,7 @@ static const char *ext_feature_name[] = {
     NULL, "cx16", "xtpr", NULL,
     NULL, NULL, "dca", "sse4.1|sse4_1",
     "sse4.2|sse4_2", "x2apic", NULL, "popcnt",
-    NULL, "aes", NULL, NULL,
+    NULL, "aes", "xsave", "osxsave",
     NULL, NULL, NULL, "hypervisor",
 };
 static const char *ext2_feature_name[] = {
-- 
1.7.4.4

